test/ruby_lsp_rails/runner_client_test.rb

```diff
@@ -87,17 +87,20 @@ class RunnerClientTest < ActiveSupport::TestCase
         junk = %{\nputs "1\r\n\r\nhello"}
         File.write("test/dummy/config/application.rb", content + junk)
 
+        outgoing_queue = Thread::Queue.new
+
         capture_subprocess_io do
-          outgoing_queue = Thread::Queue.new
           client = RunnerClient.create_client(outgoing_queue)
-
-          response = T.must(client.model("User"))
-          assert(response.key?(:columns))
-        ensure
-          T.must(outgoing_queue).close
+          response = client.model("User")
```

Comment:
This test was passing, but the behaviour is actually always incorrect. `client.model` was raising `EmptyMessageError` before, which means we never got to the point of asserting anything and the ensure block saved the test from erroring.

I'm investigating why this test breaks.